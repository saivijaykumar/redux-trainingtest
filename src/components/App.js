import React, { Component } from 'react';

import '../index.css'
import CommentBox from './CommentBox';
import CommentList from './CommentList';

const App=()=>(
    <div>
        <h1 className="box">React -redux with testing</h1>
        <div className="main">
        <CommentBox></CommentBox>
        <CommentList></CommentList>
        </div>
    </div>
)

export default App;