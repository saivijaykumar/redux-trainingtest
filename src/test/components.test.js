import React from 'react';
import renderer from 'react-test-renderer';
import CommentList from '../components/CommentList';
import App from '../components/App';

describe('components',function(){
    describe('<App/>',function(){
        it('reneders correctly',function(){
            var domtree=renderer.create(<App></App>).toJSON();
            expect(domtree).toMatchSnapshot();
        });
    });
});