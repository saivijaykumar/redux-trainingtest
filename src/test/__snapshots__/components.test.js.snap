// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <App/> reneders correctly 1`] = `
<div>
  <h1
    className="box"
  >
    React -redux with testing
  </h1>
  <div
    className="main"
  >
    <div />
    <div />
  </div>
</div>
`;
